app/service/rest.py

```diff
@@ -530,11 +530,28 @@ def search_for_notification_by_to_field(service_id, search_term, statuses, notif
         page=1,
         page_size=current_app.config['PAGE_SIZE'],
     )
+
+    # We try and get the next page of results to work out if we need provide a pagination link to the next page
+    # in our response if it exists. Note, this could be done instead by changing
+    # notifications_dao.dao_get_notifications_by_recipient_or_reference to be True which will enable us to use
```

Comment:
> this could be done instead by changing `notifications_dao.dao_get_notifications_by_recipient_or_reference` to be True
    
What does changing `notifications_dao.dao_get_notifications_by_recipient_or_reference` to True mean?
Do you mean it could return true? That would be a totally different function then, wouldn't it?

Reply:
Good point Pea, this was inaccurate. I've updated the docs there.